job/label_sync_cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1beta1
```

Comment:
Let's check our VKE version. Should it be v1beta1 or v1?

Reply:
Sorry... This piece of code is copied directly from upstream... I didn't check the correctness of it...

I check the VKE kubernetes version is `v1.20.12`, I'm not sure it's v1 or v1beta.